horse/web/app.py

```diff
@@ -1,8 +1,8 @@
 from flask import Flask
 
-from .recommendations import recommendations_bp
-from .movies import movies_bp
-from .users import users_bp
+from horse.web.movies import movies_bp
+from horse.web.recommendations import recommendations_bp
+from horse.web.users import users_bp
```

Comment:
now, if everything is located in a single package than there's even more reason to use relative imports 😄